src/client/components/builder/QuestionsTab.tsx

```diff
@@ -24,41 +24,42 @@ import { BuilderActionEnum, ConfigArrayEnum } from '../../../util/enums'
 
 const TITLE_FIELD_INDEX = -1
 
-const defaultNumericField: checker.Field = {
-  id: 'N',
+const generateDefaultNumericField = (id: number): checker.Field => ({
+  id: `N${id}`,
```

Comment:
The PR looks good - could we check with Nat if she's okay with these identifiers (`N`, `R`, etc)?

Reply:
The identifiers used are okay - just need to make changes if any field requires the same letter in the future